data/lua/wml-tags.lua

```diff
@@ -967,7 +967,9 @@ function wml_actions.terrain_mask(cfg)
 		rules[#rules + 1] = rule
 	end
 	if cfg.mask_file then
-		mask = wesnoth.read_file(cfg.mask_file)
+		local resolved = filesystem.resolve_asset(filesystem.asset_type.MAP, cfg.mask_file)
```

Comment:
Mainline seems inconsistent about whether map-masks should be in a "maps" or a "masks" folder. SoF uses "maps", while Liberty, LoW and HttT use "masks".

I think "masks" makes sense, but please let's move this to a separate PR where we can discuss conventions and whether the editor needs changes at leisure, without needing to affect the timing of the reorg PR.

Reply:
I have the opposite opinion; a mask is a type of map, so I think it makes more sense to put them in the same folder. There's always the option of putting them in a subfolder if you really want them separate.